apps/web-portal/lib/WebPortalExpressApp.mjs

```diff
@@ -44,8 +45,14 @@ export class WebPortalExpressApp {
   }
 
   startAndAttach() {
```

Comment:
plz, add jsdocs

Reply:
done

Reply:
it's not a property, provide jsdocs like for method

For example:

```
  /**
   * Starts all nessesary application's component and attach it's execution to current process
   * @return {undefiend} 
   */

```